### DISCUSSION TITLE
Hydration Error With Day.JS

### DISCUSSION THREAD

**Discussion by Geometrically** [nuxt/nuxt#18777] [Questions] [answered]
Hey,

I am using the `dayJs(date).fromNow()` plugin to display pretty dates like `14 minutes ago` or `5 minutes ago`. This works fine in a lot of cases, but occasionally, when Nuxt renders the page server side and when it renders client side, it is in between the minutes gap, causing a hydration error.

Ex: A hydration error can occur for `15 minutes ago` on the server and `16 minutes ago` on the client, if 1 second elapsed between the initial server render and the client side hydration.

How should I fix this? Should I store the currentDate in a state? It seems a bit silly- is there a better workaround?

Thanks!

**Comment by TheAlexLichter** [comment, marked as answer]
Yes - you either can:

*  utilize `useState` to create the value on the server-side and "transfer" it to the client-side
* generate it on the client-side only.
